changelogs/fragments/1542-vmware_cluster_drs.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - vmware_cluster_drs - Add predictive DRS Setting and invert the drs_vmotion_rate => same like in the vcenter (https://github.com/ansible-collections/community.vmware/pull/1542).
```

Comment:
```suggestion
  - vmware_cluster_drs - Add predictive DRS Setting (https://github.com/ansible-collections/community.vmware/pull/1542).
bugfixes:
  - vmware_cluster_drs - Fix drs_vmotion_rate so that the specified value is configured correctly.
    Previously, choosing 1 / 2 configured a migration threshold of 5 / 4 and vice versa (https://github.com/ansible-collections/community.vmware/pull/1542).
```